### ISSUE TITLE
Support running on URL with path prefix

### ISSUE THREAD

**Issue by joschi** [NicolasConstant/sengi#355]
Thanks for making Sengi and keeping it alive!

While the primary use case doesn't seem to be hosting it on one's web server, it would be great if it supported running with a URL path prefix.

Right now it assumes that it's running in the "root" of a domain (`https://example.com/`) and doesn't work if it's running with a path prefix (`https://example.com/sengi/`).

This would also enable running Sengi on GitHub pages without using a custom CNAME. 😄

**Comment by NicolasConstant** [comment]
Yes it's an issue I have with the PWA aspect of it, I didn't succeed to make it work the way I wanted. If you have your own fork I could show you what to change to make it work in your use case, but on my side I just stopped trying to make it work on every possible use cases.
